### ISSUE TITLE
Feature Request: Make TabPanel-Header color costumizable

### ISSUE THREAD

**Issue by fpue** [rjaros/kvision#184]
First off, great work! I like this project and see a lot of potential here.
For my request: As the headline says, it would be nice to be able to change the color of the TabPanel-Header. The Grey/Blue-pattern just does not fit to every project.
Thanks in advance!

**Comment by rjaros** [comment]
Hello and thx for your interest in KVision :)

I can change access on `link` component inside `Tab` container from `internal` to `public`. This way you will be able to style it with css properties (e.g. color, background) and css classes, like this:

```
tabPanel {
    tab("My tab") {
        link.colorName = Col.RED
        link.addCssClass("mytab")
        div("Some content")
    }
}
```
As for the appearance itself, the current one is simply imposed by Bootstrap and changing it is not an easy task. But you can find some styling guidance on stackoverflow:
https://stackoverflow.com/questions/51701512/how-to-change-the-color-of-active-tab-in-bootstrap/51701681

**Comment by fpue** [comment]
That will be very helpfull in the future, thanks a lot for the fast response and action!

**Comment by rjaros** [comment]
Fixed in 3.16.0
